src/main/java/com/fasterxml/jackson/databind/deser/std/StdValueInstantiator.java

```diff
@@ -135,6 +135,9 @@ public void configureFromObjectSettings(AnnotatedWithParams defaultCreator,
         _delegateArguments = delegateArgs;
         _withArgsCreator = withArgsCreator;
         _constructorArguments = constructorArgs;
+        if (_defaultCreator == null) {
+            _defaultCreator = withArgsCreator;
+        }
```

Comment:
Same idea here. `create()` method should become the defaultCreator for Builder case. Idk how zero-arg `create()` method became `withArgsCreator` probably null-checking but not checking length?

Reply:
I also do not quite understand intended logic here: `_defaultCtor` is specifically expected to be 0-args Constructor; here no checking is done.
If such logic for defaulting was needed, it should be probably done by caller, not this method.